+incf/+livescripts/setup.m

```diff
@@ -39,8 +39,8 @@
 
     switch from_location
         case "online"
-            % TODO
-
+            msgbox("Unfortunately, MATLAB Online is not yet automated." + newline + ...
```

Comment:
Overall, looks good to me. My only suggestion would be to replace "Unfortunately, " with "At this time, ". 
This is a more matter-of-fact style.